nntrainer/compiler/ini_interpreter.cpp

```diff
@@ -118,26 +128,120 @@ std::shared_ptr<Layer> section2layer<PlainLayer>(dictionary *ini,
 }
 
 template <>
-std::shared_ptr<Layer> section2layer<BackboneLayer>(dictionary *ini,
-                                                    const std::string &sec_name,
-                                                    const AppContext &ac) {
-  return nullptr;
-}
+std::shared_ptr<Layer>
+section2layer<BackboneLayer>(dictionary *ini, const std::string &sec_name,
```

Comment:
This is mostly copy of `model_loader::loadBackboneConfigExternal` + `model_loader::loadBackboneConfigIniCommon`

Reply:
The original function is being removed in upcoming PR